front/src/components/RegistrationUser/RegistrationUser.component.css

```diff
@@ -0,0 +1,30 @@
+.close {
+  position: absolute;
+  left: 88.50%;
+  top: 11px;
+}
+.btn_padding {
+  letter-spacing: 1px;
+}
+.item {
+height: 95px;
+}
+.text {
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 144%;
+  align-items: flex-end;
+  text-align: center;
+  color: #292c33;
+}
+.entrance {
+  font-weight: 600;
+  padding: 30px;
+  font-size: 18px;
+}
+.accountCreate{
+  text-align: center;
+  margin: 15px;
+  line-height: 2.0;
+  height: 450px;
+}
```

Comment:
incorrect file name. please read guide https://github.com/horba/film-finity/wiki/Frontend-Namings-and-Structure
components/registration-dialog/registration-dialog.component.css
components/registration-dialog/registration-dialog.component.html
components/registration-dialog/registration-dialog.component.js
components/registration-dialog/registration-dialog.component.vue